aeneas/src/arm64/Arm64Backend.v3

```diff
@@ -49,43 +42,58 @@ class Arm64Backend extends MachBackend {
 		super(compiler, prog, mach, w) {
 
 		asm = Arm64MacroAssembler.new();
-		if (ri_gc != null) {
-			// call the RiRuntime.gc() method
-			prog.ERROR.unimplemented();
-		} else {
-			// there is no appropriate RiRuntime.gc() method
-			objReg = Arm64Regs.R0;
-			sizeReg = Arm64Regs.R0;
-		}
+
 		codegen = SsaArm64Gen.new(context, mach, asm, w, dwarf);
-		if (compiler.useGlobalRegAllocMatcher != VstMatcher.None) allocateRegsGlobal = GlobalRegAlloc.new(Regs.SET, codegen).allocate;
-		if (compiler.LocalRegAlloc) allocateRegs = LocalRegAlloc.new(Regs.SET, codegen).allocate;
-		else allocateRegs = SimpleRegAlloc.new(Regs.SET, codegen).allocate;
+		if (compiler.useGlobalRegAllocMatcher != VstMatcher.None) allocateRegsGlobal = GlobalRegAlloc.new(MRegs.SET, codegen).allocate;
+		if (compiler.LocalRegAlloc) allocateRegs = LocalRegAlloc.new(MRegs.SET, codegen).allocate;
+		else allocateRegs = SimpleRegAlloc.new(MRegs.SET, codegen).allocate;
 	}
 
 	// Override MachBackend	
+
 	def genEntryStub() {
-		prog.ERROR.unimplemented();
+		mach.fail("not implemented");
```

Comment:
I think it'd be helpful to introduce a method that is `unimplemented()` to save a little redundancy.